Source/WebCore/css/html.css

```diff
@@ -23,6 +23,11 @@
 
 @namespace "http://www.w3.org/1999/xhtml";
 
+:root {
```

Comment:
this is modifying a file shared with other platforms, has it been discussed with upstream on webkit bugtracker? To make sure we are doing this correctly

Reply:
Generally for me it's better if all changes to common files are upstreamed, then I have no conflicts when merging changes in our branch. It's not a reason to block this change, but if the plan is to have this as a patch here forever, I prefer to know about it. Or maybe if WebKi devs can explain how they do it in other platforms and we can see if it's possible to reuse the same way they do?